app/javascript/src/components/Dashboard/Contacts/constants.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+import { convertDateToStdFormat } from "utils/dayjs 4.31.02 PM";
+
+import Avatar from "./Table/Avatar";
+import RowActions from "./Table/RowActions";
+
+// Start of Contact Table component constants -------------------------
+export const DUMMY_CONTACT_DATA = {
+  name: "Ronald Reagan",
+  email: "ronald@gmail.com",
+  createdAt: convertDateToStdFormat(new Date()),
+};
+
+// code to quickly generate contact array to test Note Listing UI, may experience bugs due to non unique key gen.
+export const ROW_DATA = Array(100).fill(DUMMY_CONTACT_DATA);
+
+export const COLUMN_DATA = [
+  {
+    title: "NAME & ROLE",
+    dataIndex: "name",
+    key: "name",
+    width: "33%",
+    // eslint-disable-next-line react/jsx-filename-extension
```

Comment:
This is because in the very next line I have added a line :`render: name => <Avatar name={name} />` this causes a conflict because the line itself is 'jsx' whereas the file is '.js' . I could just change the name of the file or I could just move the column data declaration to another file, but either of them seems overkill and I am not sure moving the column data to another file does indeed follow react best practices. If any changes are needed please do inform me.

Reply:
I get your point.

In such cases you can move this into the utils file and rename this as a util function. It's actually better to move this into a util function. Because we might've to pass in some args and dynamically build the table column data when the app grows. You can call this util function buildContactTableColumns